.github/workflows/schema-check.yml

```diff
@@ -4,8 +4,11 @@ on:
   pull_request:
     branches:
       - main
+<<<<<<< HEAD
+=======
     paths:
       - "**.ifc"
+>>>>>>> refs/remotes/origin/Development
```

Comment:
We have merge conflicts' git's syntax committed here. Please revert such additions / delete such added lines. If you need help, we can have a quick call.

Reply:
Tell me when you are free to have a call

Reply:
Looking at #67 - has this PR been superseded and can thus be closed? Perhaps we don't even need to resolve this problem.